src/components/RecipeCard.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { TimeIcon } from '@chakra-ui/icons';
+import Img from 'gatsby-image';
+import { graphql } from 'gatsby';
+
+interface Props {
+  data: GatsbyTypes.RecipeCardFragment;
+}
+
+function RecipeCard({ data }: Props): JSX.Element {
+  return (
+    <div style={{padding: '50px 10px 50px 10px', marginRight: '25px'}}>
+      <div style={{ marginBottom: '5%'}}>
+        <h3
+          style={{
+            position: 'absolute',
+            fontWeight: 850,
+            textAlign: 'left',
+            fontSize: 14,
+          }}
+        >
+          {data.title}
+        </h3>
+        <TimeIcon
+          style={{ marginTop: 3, color: '#65BF73', position: 'absolute', marginLeft: '78%'}}
```

Comment:
This positioning causes the time icon to move unexpectedly:

I get this at width 1024px:
![image](https://user-images.githubusercontent.com/8410924/109534484-6182c380-7a89-11eb-8297-d1c598e81013.png)

I would recommend placing the icon and the minute text in their own right-aligned div